dnaStreaming/customer_config.json

```diff
@@ -1,7 +1,7 @@
 {
 
   "user_key": "",
-  "user_id": "",
```

Comment:
Shouldn't we keep this for the service account auth?

Reply:
Oh, yes, good catch. I deleted the wrong field. I meant to delete "user_key" so that this file would only have "service_account_id", since that is what's documented in the README for this repo.

The JS repo however documents user_key. Is this a problem? Should they both document the same param and if so what should it be? When you worked personally with the customer earlier this week did you direct them to use "service_account_id" or "user_key"?